core/include/reboot.h

```diff
@@ -28,7 +30,10 @@
  *
  * This function is used by core_panic() when the DEVELHELP macro is not defined.
  */
-void reboot(void);
+static inline void reboot(void)
+{
+    pm_reboot();
+}
```

Comment:
Yes. I kept the interface for now in order to keep the PR small, and to maybe, have the option to hook in things, like a message, at a central spot.